test/migrators/division/never_division.hrx

```diff
@@ -7,7 +7,17 @@
   f: (bold 6px/3px sans-serif);
   g: (six / three);
   h: 6 / three;
+  i: #{$x} / #{6px / 3px} / #{2};
 }
 
-<==> log.txt
-Nothing to migrate!
+<==> output/entrypoint.scss
+:foo(#{slash-list(6, 3)}) {
+  b: slash-list(6px, 3px);
```

Comment:
Actually, looking at these examples, they're really hard on the eyes. After this lands, can we make the migrator smart enough to only migrate to `slash-list()` in a non-plain-CSS context? Basically, we should generate `slash-list()` only if either `_isDivisionAllowed` is `true` *or* one of the arguments (possibly deeply-nested) is an interpolation expression.